dotnet/src/Extensions/Planning.StepwisePlanner/SystemStep.cs

```diff
@@ -26,7 +26,7 @@ public class SystemStep
     /// Gets or sets the variables for the action
     /// </summary>
     [JsonPropertyName("action_variables")]
-    public Dictionary<string, string>? ActionVariables { get; set; }
+    public Dictionary<string, object>? ActionVariables { get; set; }
```

Comment:
>public Dictionary<string, object>? ActionVariables { get; set; } [](http://example.com/codeflow?start=4&length=64)

why do we need to change that from string to object? I'm worried to diverge too much from the SK repo since at some point I'd like to move our solution to use the nuget package of SK

Reply:
I believe I changed this after running into some serialization issues with the playfab api body. I'll see if I can get it working without these changes

Reply:
We can also create our own planner